src/libs/auth.js

```diff
@@ -259,25 +259,37 @@ authStore.subscribe(withErrorIgnoring(async (state, oldState) => {
 
 authStore.subscribe((state, oldState) => {
   if (state.nihStatus !== oldState.nihStatus) {
-    const notificationId = 'nih-link-warning'
     const now = Date.now()
     const expireTime = state.nihStatus && state.nihStatus.linkExpireTime * 1000
-    const expireStatus = Utils.cond(
-      [!expireTime, () => null],
-      [now >= expireTime, () => 'has expired'],
-      [now > expireTime - (1000 * 60 * 60 * 24), () => 'will expire soon']
-    )
-    if (expireStatus) {
+    const shouldNotify = expireTime && now > expireTime - (1000 * 60 * 60 * 24)
+    if (shouldNotify) {
+      const hasExpired = now >= expireTime
+      const notificationId = hasExpired ? 'nih-link-expired' : 'nih-link-expires-soon'
```

Comment:
Split this into two notification IDs so that the expired notification will still be shown if the expires soon notification is muted.